resources/js/pages/admin/dashboard/recursosHumanos/tabs/InformacionPersonal.vue

```diff
@@ -1,40 +1,956 @@
 <template>
-    <div>
-        <div class="flex flex-wrap items-center">
-            <h1 class="text-xl text-gray-500 pl-5 mr-5 font-bold"><em class="fas fa-file-signature"></em> Registro de
-                Informacion</h1>
-        </div>
+    <div class="p-6 bg-gray-100">
+        <form @submit.prevent="handleSubmit">
```

Comment:
No debería ir así, como en las demás tabs?
```suggestion
        <form @submit.prevent="handleSubmit" class="bg-white rounded-lg shadow p-6">
```